app-i18n/libime/libime-1.1.3.ebuild

```
@@ -3,51 +3,34 @@
 
 EAPI=8
 
-inherit cmake xdg-utils flag-o-matic toolchain-funcs
+inherit cmake
 
-KEYWORDS="~amd64 ~arm64 ~loong ~riscv ~x86"
-SRC_URI="https://download.fcitx-im.org/fcitx5/libime/libime-${PV}_dict.tar.xz"
 DESCRIPTION="Fcitx5 Next generation of fcitx "
 HOMEPAGE="https://fcitx-im.org/"
+SRC_URI="https://download.fcitx-im.org/fcitx5/libime/libime-${PV}_dict.tar.xz"
+
 LICENSE="LGPL-2+"
 SLOT="5"
-IUSE="coverage doc test"
+KEYWORDS="~amd64 ~arm64 ~loong ~riscv ~x86"
+IUSE="doc test"
 RESTRICT="!test? ( test )"
+
 RDEPEND="
 	>=app-i18n/fcitx-5.1.5:5
 	app-arch/zstd:=
 	dev-libs/boost:=
-	doc? ( app-doc/doxygen )
-"
-DEPEND="
-	${RDEPEND}
-	test? (
-		coverage? (
-			dev-util/lcov
-		)
-	)
 "
+DEPEND="${RDEPEND}"
 BDEPEND="
 	kde-frameworks/extra-cmake-modules:0
 	virtual/pkgconfig
+	doc? ( app-doc/doxygen )
```

`+doc`好像什么都不会发生，doxygen要启用`dot`
-- Found Doxygen: /usr/bin/doxygen (found version "1.9.8") found components: doxygen missing components: dot
```suggestion
	doc? ( app-doc/doxygen[dot] )
```

---

估计好多包都要修这个....

---

https://bugs.gentoo.org/430914

---

吓得我赶紧检查了fcitx核心包，幸亏没有 doc use

---

这里的 This comment was marked as resolved.  是你点的么？
看起来的确是应该加 	doc? ( app-doc/doxygen[dot] ) 呀

---

app-i18n/fcitx:5也有....
但是我测试的时候发现，fcitx，libime在app-doc/doxygen[dot] 和-DENABLE_DOC=yes还是不会执行doxygen生成${BUILD_DIR}/doc/{html,latex}
不知道是哪里出了问题，就想着算了

---

那我得先看看了。我看漏了fcitx:5

---

应该解决了。你也帮忙看看？

---

latex的doc似乎不太好处理

---

然后就像b.g.o#430914里面说的，用`app-doc/doxygen` ` media-gfx/graphviz[svg]`代替`app-doc/doxygen[dot]` 应该就好了

---

更新了